paasta_tools/marathon_tools.py

```diff
@@ -658,16 +659,28 @@ def format_marathon_app_dict(self) -> FormattedMarathonAppDict:
 
         net = get_mesos_network_for_net(self.get_net())
 
-        complete_config: FormattedMarathonAppDict = {
-            'container': {
+        if self.get_gpus() is None:
```

Comment:
Even though this is one line, can you extract this out into a `get_container_type()` function?
I promise it will be useful in the future. No need for a config at this point, just make that function do the gpu logic at at first.